scripts/start

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env node
+
+require('dotenv').config()
+
+const version = require('../package.json').version
+const execute = require('child_process').execSync
+
+let build
+// Build number from the short hash
+if (process.env.NOW_GITHUB_COMMIT_SHA) {
```

Comment:
I think we can remove the now and Github Action env vars here, as we should never be using the `start` command in those environments.